homeassistant/components/zwave_js/device_trigger.py

```diff
@@ -458,33 +484,19 @@ async def async_get_trigger_capabilities(
         CENTRAL_SCENE_VALUE_NOTIFICATION,
         SCENE_ACTIVATION_VALUE_NOTIFICATION,
     ):
-        if value.metadata.states:
-            value_schema = vol.In({int(k): v for k, v in value.metadata.states.items()})
-        else:
-            value_schema = vol.All(
-                vol.Coerce(int),
-                vol.Range(min=value.metadata.min, max=value.metadata.max),
-            )
+        value_schema = get_value_state_schema(value)
 
         return {"extra_fields": vol.Schema({vol.Optional(ATTR_VALUE): value_schema})}
```

Comment:
What happens if the value schema is `None`?

Reply:
It won't be in this case because config values are the only ones that can have a schema of `None` and this path will never have a config value. With that being said I can add a guard for that just in case